src/transformers/modeling_utils.py

```diff
@@ -1747,6 +1747,9 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 quantization works well for values of magnitude ~5, but beyond that, there is a significant performance
                 penalty. A good default threshold is 6, but a lower threshold might be needed for more unstable models
                 (small models, fine-tuning).
+            no_load_in_8bit_modules (`List[str]`, *optional*, defaults to `None`):
```

Comment:
I'd definitely wait for @sgugger before merging this one, as this is a core API.

I'm not sure `no_load_in_8bit_modules` is very intuitive.  

I'd use `load_in_8bit` prefix for all 3 arguments, and then have the post_fix differentiator:

```
load_in_8bit
load_in_8bit_threshold
load_in_8bit_skip_modules
```